<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgToolOptions</class>
 <widget class="QWidget" name="WdgToolOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>196</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="bnAddReferenceImage">
     <property name="text">
      <string>Add reference image...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bnLoad">
     <property name="text">
      <string>Load set...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bnSave">
     <property name="text">
      <string>Save set...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bnDelete">
     <property name="text">
      <string>Delete all reference images</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkKeepAspectRatio">
     <property name="text">
      <string>Keep aspect ratio</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KisShapePropertySlider" name="opacitySlider" native="true"/>
   </item>
   <item>
    <widget class="KisShapePropertySlider" name="saturationSlider" native="true"/>
   </item>
   <item>
    <widget class="QRadioButton" name="radioEmbed">
     <property name="text">
      <string>Embed to .kra</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioLinkExternal">
     <property name="text">
      <string>Link to external file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisShapePropertySlider</class>
   <extends>QWidget</extends>
   <header>KisSelectionPropertySlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
